.github/workflows/Build-And-Test.yml

```diff
@@ -85,6 +85,57 @@ jobs:
         install: >-
           mingw-w64-x86_64-toolchain
 
+    # As of Nov 8, 2021, mingw-w64-x86_64-toolchain-11.1.x
```

Comment:
># [](http://example.com/codeflow?start=4&length=1)

Is there a different GDB that we should use that is consistent but also supports the exception stuff that you wanted?

Reply:
10.2 should support `-catch-throw`. The version I see that dosent support it is `8.1`.